### ISSUE TITLE
why use alpha/r in stead of alpha?

### ISSUE THREAD

**Issue by dingguo1996** [microsoft/LoRA#101]
Paper said we need scale the lora learning rate with alpha/r. But why use alpha/r in stead of alpha?
![image](https://github.com/microsoft/LoRA/assets/22886020/f3e65253-c773-46c8-b5a1-d25c5cf6b329)

**Comment by edwardjhu** [comment]
The magnitude of the preactivation after B is \Theta(r) after training with adaptive optimizers. Dividing by r stabilizes it and makes HP tuning easier as mentioned at the end of the paragraph.

**Comment by chrisway613** [comment]
> The magnitude of the preactivation after B is \Theta(r) after training with adaptive optimizers. Dividing by r stabilizes it and makes HP tuning easier as mentioned at the end of the paragraph.

Thx! 
But I want to further know about "The magnitude of the preactivation after B is \Theta(r)", would you like to show us an explanation? @edwardjhu
